Examples/Scripts/Python/truth_tracking_gsf_refitting.py

```diff
@@ -87,18 +84,28 @@ def runRefittingGsf(
             inputTracks="tracks",
             inputParticles="truth_seeds_selected",
             inputTrackParticleMatching="track_particle_matching",
-            filePath=str(outputDir / "performance_refitter.root"),
+            filePath=str(outputDir / "performance_gsf_refit.root"),
         )
     )
 
     return s
 
 
 if __name__ == "__main__":
+    srcdir = Path(__file__).resolve().parent.parent.parent.parent
     outputDir = Path.cwd()
 
     # detector, trackingGeometry, decorators = getOpenDataDetector()
+    # digiConfigFile = (
+    #     srcdir / "thirdparty/OpenDataDetector/config/odd-digi-smearing-config.json"
+    # )
```

Comment:
```suggestion
    # # ODD
    # from acts.examples.odd import getOpenDataDetector
    #
    # detector, trackingGeometry, decorators = getOpenDataDetector()
    # digiConfigFile = (
    #     srcdir / "thirdparty/OpenDataDetector/config/odd-digi-smearing-config.json"
    # )
```

One line above also needs to be substituted.